modules/ocl-loader/src/main/resources/resources.yaml

```
@@ -0,0 +1,76 @@
+resources:
+  - category: vpc
```

```suggestion
  - category: kind
```
How about replacing the `category` with `kind`?